readme.md

```diff
@@ -115,6 +116,15 @@ erDiagram
         int answer_id FK
         int user_id FK
     }
+
+    user_task_days ||--|| users : user_id
```

Comment:
```suggestion
    user_task_days }o--|| users : user_id
```
We have userTaskDays relation as `Model.HasManyRelation` in user model. Single user can have from 0 user_task_days created to multiple